geminidr/core/primitives_preprocess.py

```diff
@@ -533,6 +501,147 @@ def dilateObjectMask(self, adinputs=None, suffix=None, dilation=1, repeat=False)
             gt.mark_history(ad, primname=self.myself(), keyword=timestamp_key)
         return adinputs
 
+    def fixPixels(self, adinputs=None, suffix=None, axis=None, regions=None,
+                  regions_file=None, use_local_median=False, debug=False):
+        """
```

Comment:
Some of the other primitives in this file also have the explicit list (`darkCorrect`, `flatCorrect`, ...). I find this much more natural instead of having to unpack manually the params dict later. Btw the other primitives also use `suffix=None` which seems better than defining the real default value in multiple places (since it could change in the parameters module and get out of sync). Since the params are validated upstream the default value here does not matter, but at least it's good to have the explicit list in the signature instead of having the read the full code to find all the parameters a primitive can have (and docstrings are frequently incomplete).

Reply:
Specifying parameters in the call signature has been used sparingly when there aren't many and there's no chance of them being different for different instruments or changing. In many cases, `**params` is useful because many of the parameters are passed to a helper function (such as `fit_1D`) but it's just a style thing.